about.html

```diff
@@ -0,0 +1,191 @@
+          />
+        </defs>
+      </svg>
+      <img
+        class="header__img"
+        src="./img/logo.png"
+        alt="CC Global Summit 2015"
+      />
+    </section>
+    <section class="hero__about"></section>
+
+    <main class="main">
+      <section class="main__section">
+        <h2 class="main__section-h2">
+          Creative Commons Global Summit 2015 Logo
+        </h2>
+
+        <svg
+          class="main__section-svg"
+          xmlns="http://www.w3.org/2000/svg"
+          width="31"
+          height="3"
+          viewBox="0 0 31 3"
+          fill="none"
+        >
+          <line
+            x1="0.5"
+            y1="1.5"
+            x2="30.5"
+            y2="1.5"
+            stroke="#A04038"
+            stroke-width="2"
+          />
+        </svg>
+
+        <p class="main__section-p">
+          The logo of CC Global Summit 2015 was decided through the logo
+          competition from 8, June to 7, July.
+        </p>
+        <img
+          class="main__section-img"
+          src="./img/about_logo_2.png"
+          alt="Company logo"
+        />
+      </section>
+      <section class="main__articles">
+        <h2 class="main__articles-h2">See the past CC Global Summits</h2>
+        <svg
+          class="main__section-svg"
+          xmlns="http://www.w3.org/2000/svg"
+          width="31"
+          height="3"
+          viewBox="0 0 31 3"
+          fill="none"
+        >
+          <line
+            x1="0.5"
+            y1="1.5"
+            x2="30.5"
+            y2="1.5"
+            stroke="#A04038"
+            stroke-width="2"
+          />
+        </svg>
+        <p class="main__articles-p">
+          Take a look at the past two CC Global Summits which took place in
+          Buenos Aires and in Warsaw
+        </p>
+        <div class="article-wrapper">
+          <article class="main__article">
+            <h2 class="main__article-h2">2011</h2>
+            <p class="main__article-p">CC Global Summit 2011 in Warsaw</p>
+          </article>
+          <article class="main__article">
+            <h2 class="main__article-h2">2011</h2>
+            <p class="main__article-p">CC Global Summit 2011 in Warsaw</p>
+          </article>
+        </div>
```

Comment:
- [ ] In this part of your about section, kindly ensure that the images you used are overlayed to look like the images used in the original design so that the text in the images will be more legible :100: 
- [ ] KIndly increase the size of the images as well and ensure that their layout is similar to that of the original design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/226072158-5a81618b-4fc7-424d-99d9-d7dad8483447.png)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/226072133-80398282-2f1f-453e-9bce-78e8b121d4c0.png)